account_move_helper/README.rst

```diff
@@ -14,28 +14,25 @@
 Journal Entry Helper
 ====================
 
-This module adds two buttons in journals entries to allow you to set required balance for accounts and it will automatically create the move line for you
+This module adds two buttons in journals entries to allow you to set required balance for accounts and it will automatically create the move line for you.
 
 Installation
 ============
 
```

Comment:
If nothing to say please remove the content of the section, leave the section tittle in order to follow guideline.
apply here in also in the rest of the rst file.